### PR TITLE
Build of updates from PR 12 in ISMMS repo

### PR DESCRIPTION
**Pull request by rando2**
- Submitted minor changes in https://github.com/ismms-himc/covid-19_sinai_reviews/pull/12
- Followed #178 to build

### CODE REVIEW

**Comment by rando2** [comment]
Oh no, my local version is out of sync with the master because of trying to demo the hubs. I need to fix this to remove the R code before we can merge.

**Comment by agitter** [comment]
We should merge #178 before finalizing this to test that the upstream commit hash is updated in the appendix.

**Comment by rando2** [comment]
> We should merge #178 before finalizing this to test that the upstream commit hash is updated in the appendix.

That sounds good to me @agitter ! I wasn't sure if you want to wait for Casey's approval on merging that.

**Comment by agitter** [comment]
I merged #178 so you should be able to pull that version of the script and test it here.

**Comment by rando2** [comment]
OK, I guess there is one thing in #178 that isn't quite clear to me -- not sure what should I be hashing to check if my hash is right!

File: content/95-ismms-himc-appendix.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/b8d8df55b4f2416da4ca3b7c765c8fff48acf5e3 -->
+<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/05d387bf10b829dad6bedb4bf313686750b78cc9 -->
```
Comment (agitter) [inline content/95-ismms-himc-appendix.md:1]: @rando2 you don't have to hash anything yourself.  My script update in #178 updates this line in the appendix when you re-run the script.  It gets the latest commit from https://github.com/ismms-himc/covid-19_sinai_reviews and uses that SHA-1 hash to version the appendix.  It looks like everything is working as intended.

Comment (rando2) [reply content/95-ismms-himc-appendix.md:1]: Awesome! Thanks @agitter!

**Pull request merged**